src/attestation.rs

```diff
@@ -499,7 +498,9 @@ pub async fn log_summary(
     messages_sent: Vec<Result<String, OperationError>>,
     result_strings: Vec<Result<ComparisonResult, OperationError>>,
     radio_name: &str,
-    config: &Config,
+    slack_token: &Option<String>,
```

Comment:
Do you think it would be worth creating mocked `&Config` and pass that in this function instead of separating out the fields? I was thinking to refactor the notification functions with curried configurations later on

Reply:
I don't see the need for the full config to be passed into this function 🤔